src/features/commands/unban.js

```diff
@@ -0,0 +1,45 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('unban')
+        .setDescription('Unbans a user from the server.')
+        .addStringOption(option =>
+            option.setName('snowflake')
```

Comment:
Could you change this to user instead? That would be more consistent and majority of people are unfamiliar with the term snowflake.
```suggestion
            option.setName('user')
```